backend/lambdas/custom_resources/validate_vpce_subnets.py

```diff
@@ -0,0 +1,40 @@
+import boto3
+from crhelper import CfnResource
+from decorators import with_logging
+
+helper = CfnResource(json_logging=False, log_level="DEBUG", boto_level="CRITICAL")
+
+ec2_client = boto3.client("ec2")
+
+
+@with_logging
+@helper.create
+@helper.update
+def create(event, context):
```

Comment:
There are some services that do not have VPCEs in China region and also some VPCEs for some services do not support all AZs. Therefore we needed a CR to return valid subnets. Also the domain name of the service varies depending on the service so we needed the CR to return the correct name too.

Reply:
I am not a big fan of comments in code but I think in this particular scenario, perhaps we can add a note on top of this file to explain this. Just to make sure a reader can have the necessary context to understand the reason this exists?

Reply:
fair enough, we should probably use a different name then because the goal of the CR isn't really validation, it's to retrieve some data if I've understood correctly

Reply:
Name of function changed and note added to top of file to help